generic/.gitbook.yaml

```
@@ -3,3 +3,6 @@ root: ./
 ​structure:  
     readme: README.md
     summary: SUMMARY.md
+
+redirects:
+    generic/the-licensing-model: generic/README.md
```

I don't think the site name is required, as this file is already into the generic folder.

```suggestion
    the-licensing-model: README.md
```